{"name": "test_fun3", "status": "failed", "statusDetails": {"message": "AssertionError: assert 0 == True\n  +0\n  -True", "trace": "self = <testcase.test_case01.Test_case01 object at 0x000001A12E3B7788>\n\n    @pytest.mark.run(order=2)\n    @allure.story(\"删除用户组\")\n    @pytest.mark.dependency(depends=[\"Test_case01::test_fun2\"])\n    def test_fun3(self):\n        print(\"test_func3\")\n        Test_case01.Index += 1\n        print(self.Index)\n        with allure.step(\"用户组删除操作\"):\n            payload = self.Teat.editteam(self.getdata(self.Index, self.excel1_obj.TESTDATA).get(\"payload\"))\n            print(\"fun3_payload:\", payload)\n>           assert self.Teat.deleteteam(payload)==True\nE           assert 0 == True\nE             +0\nE             -True\n\ntestcase\\test_case01.py:71: AssertionError"}, "steps": [{"name": "用户组删除操作", "status": "failed", "statusDetails": {"message": "AssertionError: assert 0 == True\n  +0\n  -True\n", "trace": "  File \"E:\\Pyitems\\DBSHOP\\testcase\\test_case01.py\", line 71, in test_fun3\n    assert self.Teat.deleteteam(payload)==True\n"}, "start": 1653227267028, "stop": 1653227291997}], "start": 1653227267028, "stop": 1653227291997, "uuid": "1362a326-68d0-4e27-830f-72c41e6655d7", "historyId": "855167528bb38eb74082a1e8a239e55f", "testCaseId": "249b4064aeaab1ea01cbc0a2ac2d0f81", "fullName": "testcase.test_case01.Test_case01#test_fun3", "labels": [{"name": "feature", "value": "用户组模块"}, {"name": "story", "value": "删除用户组"}, {"name": "tag", "value": "dependency(depends=['Test_case01::test_fun2'])"}, {"name": "tag", "value": "run(order=2)"}, {"name": "parentSuite", "value": "testcase"}, {"name": "suite", "value": "test_case01"}, {"name": "subSuite", "value": "Test_case01"}, {"name": "host", "value": "PC-20210917KHGT"}, {"name": "thread", "value": "13180-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "testcase.test_case01"}]}